toolbox/database-role-artifacts/per-project-scripts/postgrest-changes.sh

```diff
@@ -0,0 +1,128 @@
+
+# postgrest_parking
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_parking -c "GRANT USAGE ON SCHEMA api, public TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_parking -c "GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA api, public TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_parking -c "GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA api, public TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_parking -c "GRANT EXECUTE ON ALL FUNCTIONS IN SCHEMA api, public TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_parking -c "ALTER DEFAULT PRIVILEGES IN SCHEMA api, public GRANT ALL PRIVILEGES ON TABLES TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_parking -c "ALTER DEFAULT PRIVILEGES IN SCHEMA api, public GRANT ALL PRIVILEGES ON SEQUENCES TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_parking -c "ALTER DEFAULT PRIVILEGES IN SCHEMA api, public GRANT EXECUTE ON FUNCTIONS TO read_write;"
+
+# postgrest_road_conditions
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_road_conditions -c "GRANT USAGE ON SCHEMA api, public TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_road_conditions -c "GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA api, public TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_road_conditions -c "GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA api, public TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_road_conditions -c "GRANT EXECUTE ON ALL FUNCTIONS IN SCHEMA api, public TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_road_conditions -c "ALTER DEFAULT PRIVILEGES IN SCHEMA api, public GRANT ALL PRIVILEGES ON TABLES TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_road_conditions -c "ALTER DEFAULT PRIVILEGES IN SCHEMA api, public GRANT ALL PRIVILEGES ON SEQUENCES TO read_write;"
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -d postgrest_road_conditions -c "ALTER DEFAULT PRIVILEGES IN SCHEMA api, public GRANT EXECUTE ON FUNCTIONS TO read_write;"
+
+# Delete the role `ctr_user`
+docker run --rm -it --env-file ./credentials_for_throw_away_server postgres:16 psql -c "DROP ROLE IF EXISTS ctr_user;"
```

Comment:
@johnclary - thanks for your recent work removing the CTR related slice of the stack out of the postgrest stack. As [we touched on in a recent slack thread](https://austininnovation.slack.com/archives/CHZE6BC6L/p1747150834794879?thread_ts=1747150566.234719&cid=CHZE6BC6L), this is a related change to drop the associated role. Thanks!